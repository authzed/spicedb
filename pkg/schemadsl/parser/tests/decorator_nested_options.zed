definition mydefinition {
    @example(opt1, (opt2a,opt2b))
    relation foo: sometype
}