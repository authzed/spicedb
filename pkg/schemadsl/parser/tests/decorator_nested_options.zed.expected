NodeTypeFile
  end-rune = 87
  input-source = decorator nested options test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = mydefinition
      end-rune = 87
      input-source = decorator nested options test
      start-rune = 0
      child-node =>
        NodeTypeRelation
          end-rune = 85
          input-source = decorator nested options test
          relation-name = foo
          start-rune = 64
          allowed-types =>
            NodeTypeTypeReference
              end-rune = 85
              input-source = decorator nested options test
              start-rune = 78
              type-ref-type =>
                NodeTypeSpecificTypeReference
                  end-rune = 85
                  input-source = decorator nested options test
                  start-rune = 78
                  type-name = sometype
          decorator-node =>
            NodeTypeDecorator
              decorator-name = example
              end-rune = 58
              input-source = decorator nested options test
              start-rune = 30
              decorator-options =>
                NodeTypeDecoratorOptions
                  end-rune = 58
                  input-source = decorator nested options test
                  start-rune = 38
                  decorator-options =>
                    NodeTypeDecoratorOptions
                      decorator-option-value = opt1
                      end-rune = 42
                      input-source = decorator nested options test
                      start-rune = 43
                    NodeTypeDecoratorOptions
                      end-rune = 57
                      input-source = decorator nested options test
                      start-rune = 45
                      decorator-options =>
                        NodeTypeDecoratorOptions
                          decorator-option-value = opt2a
                          end-rune = 50
                          input-source = decorator nested options test
                          start-rune = 51
                        NodeTypeDecoratorOptions
                          decorator-option-value = opt2b
                          end-rune = 56
                          input-source = decorator nested options test
                          start-rune = 57